src/fondant/components/retrieve_images_from_faiss_index/fondant_component.yaml

```diff
@@ -0,0 +1,41 @@
+name: Retrieve images from FAISS index
+description: |
+ Retrieve images from a Faiss index. The component should reference a Faiss image dataset, 
+  which includes both the Faiss index and a dataset of image URLs. The input dataset consists 
+  of a list of prompts. These prompts will be embedded using a CLIP model, and similar 
+  images will be retrieved from the index.
+
+image: fndnt/retrieve_images_from_faiss_index:dev
+tags:
+  - Data retrieval
+
+consumes:
+  prompt:
+    type: string
+
+produces:
+  image_url:
+    type: string
+  prompt_id:
+    type: string
+
+previous_index: prompt_id
+
+args:
+  dataset_url:
+    description: Url of the dataset
+    type: str
+  faiss_index_path:
+    description: Url of the dataset
+    type: str
+  image_index_column_name:
+    description: Name of the column in the dataset that contains the image index
+    type: str
```

Comment:
I don't think we need this, we can just use the index of the `id_mapping`. We might want to take the `url_column_name` as a parameter, since this is not fixed in autofaiss.